api/observability/v1/output_types.go

```diff
@@ -646,6 +679,14 @@ type KafkaTuningSpec struct {
 	// +kubebuilder:validation:Enum:=none;snappy;zstd;lz4
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Compression"
 	Compression string `json:"compression,omitempty"`
+
+	// MessageTimeout messages that are waiting to be sent are dropped after this timeout.
+	// Used to ensure that newer messages are not blocked indefinitely by older ones.
+	//
+	// A duration string is a possibly signed sequence of decimal numbers, each with optional fraction
+	// and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or
```

Comment:
Remove negative example.  Do we need to say it's "signed" since only positve numbers make sense.  We should additionally add a validation to restrict it to positive numbers

Reply:
I pasted direct from the Go docs, thanks for catching.